File: Backend/src/main/java/hadi/springSecurity/models/responses/MessageBoolResponse.java
package hadi.springSecurity.models.responses;

public class MessageBoolResponse
{
	private String message;
	private boolean result;
	
	public MessageBoolResponse()
	{
	}
	
	public MessageBoolResponse(String message, boolean result)
	{
		super();
		this.message = message;
		this.result = result;
	}

	public String getMessage()
	{
		return message;
	}
	
	public void setMessage(String message)
	{
		this.message = message;
	}
	
	public boolean isResult()
	{
		return result;
	}
	
	public void setResult(boolean result)
	{
		this.result = result;
	}
}

File: Frontend/src/app/models/responses/MessageBoolResponse.ts
export class MessageBoolResponse
{
    private _message: string;
    private _result: boolean;

    constructor(message: string, result: boolean)
    {
        this._message = message;
        this._result = result;
    }

    public get message(): string
    {
        return this._message;
    }

    public get result(): boolean
    {
        return this._result;
    }

    public set message(value: string)
    {
        this._message = value;
    }

    public set result(value: boolean)
    {
        this._result = value;
    }
}